### ISSUE TITLE
Calling Formula.sha1 is disabled

### ISSUE THREAD

**Issue by kikito** [naartjie/homebrew-luajit#1]
This tap now fails with the following error:

```
❯ brew tap naartjie/luajit
❯ brew install luarocks-luajit
Error: Calling Formula.sha1 is disabled!
Use Formula.sha256 instead.
/usr/local/Homebrew/Library/Taps/naartjie/homebrew-luajit/luarocks-luajit.rb:6:in `<class:LuarocksLuajit>'
Please report this to the naartjie/luajit tap!
❯
```

**Comment by naartjie** [comment]
@kikito hi, thanks for filing.

I'm not using lua anymore. I'm not sure how to fix this error. Are there no better alternatives out there now?

**Comment by naartjie** [comment]
@kikito hi, thanks for logging this.

I'm not using lua anymore, and I'm not sure how to fix this. Aren't there better alternative out there nowadays?

**Comment by naartjie** [comment]
fixed via #2 

@kikito could you confirm it works for you?
